src/resources/js/routes.js

```diff
@@ -11,5 +12,10 @@ export const router = new VueRouter({
         path: '/hello',
         name: 'Hello',
         component: Hello
+    },
+    {
+        path: '/authentication',
```

Comment:
The path should be something like `/login`

Reply:
it's the french word /authentification now